packages/icons/src/_icons.scss

```diff
@@ -4,8 +4,7 @@
 @import "generated/icon-variables";
 @import "generated/icon-map";
 
-$pt-namespace: "pt" !default;
-$ns: $pt-namespace;
+$ns: "bp3" !default;
```

Comment:
can we not import that from core? how does it work for other packages that contain CSS?

Reply:
❌ other way around... `core` depends on `icons!

Reply:
nope no such plans. still core > icons